### PR TITLE
add phil9987 to yearbook - now with valid md file

### PR DESCRIPTION
**Pull request by phil9987**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/Phil9987/phil9987.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Philip Junker
+institution: ETH Zurich
+profile_pic: IMG_20200525_170026_078.jpg
+quote: The block of granite which was an obstacle in the path of the weak, becomes a stepping stone in the path of the strong.
```
Comment (Mohammed-Shoaib) [inline _data/Phil9987/phil9987.md:5]: Quote must **not** be longer than 100 characters, yours is **119** characters.

Comment (phil9987) [reply _data/Phil9987/phil9987.md]: Shortened in latest commit.

**Comment by phil9987** [comment]
Hi @Mohammed-Shoaib,

Thank you :)
Sorry, when I changed it last minute I didn't check again the length of the quote. Now it's 78 characters.

Have a good day!
Best,
Philip

**Comment by phil9987** [comment]
Thanks a lot for your inputs, Mohammed. I corrected the quote accordingly, I intended to write "mystery" as you suggested.
Also, I changed the foldername to phil9987.

Thank you. I wish you all the best for your current and future endeavors, too!

Best,
Philip

**Comment by Mohammed-Shoaib** [comment]
@phil9987 Thank you for making the mentioned changes! Also, thank you for your kind wishes! :heart:

Your pull request should be merged soon. Till then sit back, relax, and enjoy :relaxed:

**Pull request merged**